src/Microsoft.Azure.SignalR.Common/Endpoints/IConfigurationExtension.cs

```diff
@@ -15,14 +15,9 @@ internal static class IConfigurationExtension
         /// <remarks>
         /// The SignalR service endpoint whose key is exactly the section name is not extracted.
         /// </remarks>
-        public static ServiceEndpoint[] GetSignalRServiceEndpoints(this IConfiguration configuration, string sectionName)
+        public static IEnumerable<ServiceEndpoint> GetSignalREndpointsFromSectionChildren(this IConfiguration configuration, string sectionName)
```

Comment:
Simplify method name and it's confusing using same input/output but different method names. Suggest to reference previous way: https://github.com/Azure/azure-signalr/blob/v1.6.0/src/Microsoft.Azure.SignalR/ServiceOptionsSetup.cs#L60-L81

Reply:
Sometimes we want all the endpoints as a `IEnumerable`, if using `(string, ServiceEndoint[])`, then we have to merge the result with one more step. How about simplify the method names to `GetAllEndpoints` and `GetSuffixedEndpoints`? "SuffixedEndpoints" means endpoints whose keys have suffixes?

Reply:
Should they handle together as when loading the settings we don't distinguish them? It's internal logic to detect suffixed list or single connection string. Return `(string, IEnumerable<ServiceEndpoint>)` to save the merge?